src/lib/components/PrismicRichText.svelte

```diff
@@ -1,14 +1,38 @@
 <script lang="ts">
   import * as prismicH from "@prismicio/helpers"
-  import { usePrismic } from "./..";
+  import type * as prismicT from "@prismicio/types"
 
-  import type { RichTextField as RichTextFieldType } from "@prismicio/types";
-  import type { TitleField as TitleFieldType } from "@prismicio/types";
+  /**
+	 * A Prismic Rich Text or Title field.
+	 */
+  export let field: prismicT.TitleField | prismicT.RichTextField;
 
-  let { htmlSerializer, linkResolver } = usePrismic()
+	/**
+	 * A function that maps a Rich Text block to a React component.
+	 *
+	 * @see Learn about HTML serializers {@link https://prismic.io/docs/core-concepts/html-serializer}
+	 */
+  export let htmlSerializer: prismicH.HTMLFunctionSerializer | null | undefined = null;
```

Comment:
Is it possible to support object serializers? They are simpler to write and read than function serializers.